pkg/sdn/plugin/eventqueue.go

```diff
@@ -0,0 +1,140 @@
+package plugin
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/client/cache"
+)
+
+func NewEventQueue(keyFunc cache.KeyFunc) *cache.DeltaFIFO {
```

Comment:
it seems unlikely that the sdn code is going to be the only part of origin that wants this delta-compressing code...